examples/entry-list/src/index.js

```diff
@@ -0,0 +1,10 @@
+import { init, locations } from '@contentful/app-sdk';
+import { onEntryListUpdated } from './locations/EntryList'
+
+init((sdk) => {
+  if (sdk.location.is(locations.LOCATION_ENTRY_LIST)) {
+    sdk.entryList.onEntryListUpdated((props) => onEntryListUpdated({entries: props.entries}))
+  } else {
+    return null;
+  }
+});
```

Comment:
I guess there is no need to return `null` in case it's not `LOCATION_ENTRY_LIST`
```suggestion
init((sdk) => {
  if (sdk.location.is(locations.LOCATION_ENTRY_LIST)) {
    sdk.entryList.onEntryListUpdated((props) => onEntryListUpdated({entries: props.entries}))
  }
});
```